### ISSUE TITLE
add Science Gothic

### ISSUE THREAD

**Issue by tphinney** [google/fonts#3685]
It will need minor issues/bugs fixed first, a character set check (and likely update?), but the hope would be to have Science Gothic added late in 2021, if Google has relevant axis support in place by then.

Science Gothic is a 4-axis variable font commissioned for Google Fonts, which:
- includes axes for wght, wdth, slnt, YOPQ
- supports extended Latin (incl. Vietnamese) and extended Cyrillic
- has true small caps as well as a few other typographic niceties

Microsite: https://sciencegothic.com/
Repo: https://github.com/tphinney/science-gothic

cc @kateliev

**Comment by davelab6** [comment]
Moving to Q3 per discussion with @tphinney today

**Comment by davelab6** [comment]
Assigning to @tphinney to update here when ready for review

**Comment by chrissimpkins** [comment]
@tphinney what is the status of this project?

**Comment by tphinney** [comment]
We have been a bit distracted lately with special icon (and now icons plural) for a new Google project, as well as an unsurprising rush of new icons needed before Google I/O. I expect to be largely clear of those things in another week

There is a very large amount of less urgent but desired icon work, that I think we could prioritize lower than this. Let me confer with Dave and SeHee.

**Comment by tphinney** [comment]
OK, t conferred with Dave.

Our icon work is ALL a higher priority, including some less urgent aspects. If it is on the schedule it is lower priority. 

I can still chip at Science Gothic just when I need a break or something. But when or if it will be ready to go is unclear to me. Not Q2, highly unlikely Q3, some possibility of Q4. I could update this in six to eight months.

**Comment by chrissimpkins** [comment]
Thanks!

**Comment by tphinney** [comment]
Long-awaited update: this is inching closer, we have been chipping away at it in recent weeks. I currently expect it will be ready for onboarding review no later than September, perhaps even late August.

**Comment by davelab6** [comment]
Great news @tphinney :)

**Comment by tphinney** [comment]
Still on track for being ready for onboarding review this month.

**Comment by tphinney** [comment]
Oops, sorry, should have updated here.

We had feedback that we should reconsider the Cyrillic for more triangular forms (ironic seeing as co-designer @kateliev is Bulgarian!), and Vassil Kateliev has been doing so in between working on Google/Material Symbols.

But we are within a week or two of being ready for review, I think.

**Comment by yanone** [comment]
Tracking specific production issues: https://github.com/googlefonts/science-gothic/issues/356

**Comment by chrissimpkins** [comment]
Tentatively moving the onboarding plan to Q1 2025

**Comment by yanone** [comment]
[Discussion about the contrast axis is ongoing](https://github.com/googlefonts/axisregistry/issues/3). Before we reach a conclusion there, this issue is stalled.

**Comment by chrissimpkins** [comment]
I believe that this is now unblocked since the contrast axis registration is approved?

**Comment by yanone** [comment]
Not yet. Every two weeks we're getting an inch closer.

**Comment by davelab6** [comment]
Reassigned from @yanone  to @emmamarichal during the axis registry meeting today since he's retired. 

Also @tphinney noted there's a need for the 'build smallcaps' builder feature; and upstream doens't yet use builder, so there's work to apply builder to the repo's build system.

**Comment by davelab6** [comment]
@emmamarichal today @tphinney said this is all set for you
